INSTALLATION.md

```
@@ -6,115 +6,321 @@ Follow the instructions below to install InvoicePlane on your preferred platform
+**View Logs:**
+```bash
+docker-compose logs -f
+```
+
+**Stop Services:**
+```bash
+docker-compose down
+```
+
+**Access Points:**
+- **InvoicePlane**: http://localhost
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)